skiko/src/awtMain/cpp/windows/DCompLibrary.cc

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
It's required only for headers and should be removed from here

Reply:
Done